README.md

```diff
@@ -1 +1,153 @@
-### Webex SDK Component Adapter
+<h1 align='center' style='border-bottom: none;'>Webex SDK Component Adapter</h1>
+<h3 align='center'>Webex adapter to share SDK data with the components</h3>
+<p align='center'>
+<a href='https://circleci.com/gh/webex/sdk-component-adapter'>
+    <img alt='CircleCI' src='https://circleci.com/gh/webex/sdk-component-adapter.svg?style=shield'>
+  </a>
+  <a href='https://www.npmjs.com/package/@webex/sdk-component-adapter'>
+    <img alt='npm latest version' src='https://img.shields.io/npm/v/@webex/sdk-component-adapter?label=npm%40latest'>
+  </a>
+  <a href='#badge'>
+    <img alt='semantic-release' src='https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg'>
+  </a>
+  <a href='https://github.com/webex/sdk-component-adapter/blob/master/LICENSE'>
+    <img src='https://img.shields.io/npm/l/webex.svg' alt='license'>
+  </a>
+</p>
+
+**Webex SDK Adapter** TBD.
+
+## Table of Contents
+
+- [Install](#install)
+- [Usage](#usage)
+- [Contributing](#contributing)
+- [License](#license)
+- [Support](#support)
+- [Team](#team)
+
+## Install
+
+```bash
+npm install --save @webex/components @webex/sdk-component-adapter
+```
+
+## Usage
```

Comment:
I don't think you should include the `Styles` and `Adapters` sections here. Instead, you should link out the the Webex Components README usage section.
This usage section should only explain how to replace the JSON adapter, or any adapter with the SDK adapter.